Scripts/script-EmailAskUserResponse.yml

```diff
@@ -7,6 +7,7 @@ script: |
 
   if (res[0].Type==entryTypes.note) {
       text = res[0].Contents;
+      text = text.replace(/\n/g,'__NL__').replace(/<script .+?<\/script>/g,'').replace(/<style .+?<\/style>/g,'').replace(/__NL____NL__/g,'').replace(/__NL__/g,'\n');
```

Comment:
you handle just one new line or 2? are those the only options?
Code-wise i think you should do new assignment for each replacement for readability and can even add comments
```
text = text.replace(/<script .+?<\/script>/g,'');
text = text.replace(/<style .+?<\/style>/g,'')
...
```

Reply:
I don't agree about the new assignment for each line, that's the whole purpose of chaining... It should be wel documented.

Reply:
And I would put each replace in new line of course

Reply:
can also chain with new lines, just meant for it to be more readable

Reply:
Like this you mean?
```
    text = text.replace(/\n/g,'__NL__')
                .replace(/<script .+?<\/script>/g,'')
                .replace(/<style .+?<\/style>/g,'')
                .replace(/__NL____NL__/g,'')
                .replace(/__NL__/g,'\n');
```

Reply:
yup